common/annotations/en.xml

```diff
@@ -734,306 +734,306 @@ annotations.
+		<annotation cp="🤣">crying | face | floor | funny | haha | happy | hehe | hilarious | joy | laugh | lmao | lol | rofl | roflmao | rolling | tear</annotation>
 		<annotation cp="🤣" type="tts">rolling on the floor laughing</annotation>
-		<annotation cp="😂">crying | face | face with tears of joy | feels | funny | haha | hahaha | happy | hehe | hilarious | joy | laugh | laughing | laughter | lmao | lmfao | lol | lolol | lololol | rofl | roflmao | tear | tears</annotation>
+		<annotation cp="😂">crying | face | feels | funny | haha | happy | hehe | hilarious | joy | laugh | lmao | lol | rofl | roflmao | tear</annotation>
 		<annotation cp="😂" type="tts">face with tears of joy</annotation>
-		<annotation cp="🙂">face | happy | slightly smiling face | smile | smiling</annotation>
+		<annotation cp="🙂">face | happy | slightly | smile | smiling</annotation>
 		<annotation cp="🙂" type="tts">slightly smiling face</annotation>
 		<annotation cp="🙃">face | hehe | smile | upside-down</annotation>
 		<annotation cp="🙃" type="tts">upside-down face</annotation>
 		<!-- Generated lines from Emoji Data v14, using GenerateCldrData.java -->
-		<annotation cp="🫠">disappear | dissolve | embarrassed | haha | heat | hot | liquid | lol | melt | melting | melting face | sarcasm | sarcastic</annotation> <!-- 1FAE0 -->
+		<annotation cp="🫠">disappear | dissolve | embarrassed | face | haha | heat | hot | liquid | lol | melt | melting | sarcasm | sarcastic</annotation> <!-- 1FAE0 -->
 		<annotation cp="🫠" type="tts">melting face</annotation>
 		<annotation cp="😉">face | flirt | heartbreaker | sexy | slide | tease | wink | winking | winks</annotation>
 		<annotation cp="😉" type="tts">winking face</annotation>
-		<annotation cp="😊">blush | eye | face | glad | satisfied | smile | smiling | smiling face with smiling eyes</annotation>
+		<annotation cp="😊">blush | eye | eyes | face | glad | satisfied | smile | smiling</annotation>
 		<annotation cp="😊" type="tts">smiling face with smiling eyes</annotation>
-		<annotation cp="😇">angel | angelic | angels | blessed | face | fairy tale | fairytale | fantasy | halo | happy | innocent | peaceful | smile | smiling | smiling face with halo | spirit</annotation>
+		<annotation cp="😇">angel | angelic | angels | blessed | face | fairy | fairytale | fantasy | halo | happy | innocent | peaceful | smile | smiling | spirit | tale</annotation>
 		<annotation cp="😇" type="tts">smiling face with halo</annotation>
-		<annotation cp="🥰">adore | crush | face | heart | hearts | i love you | ily | in love | love | romance | smile | smiling face with 3 hearts | smiling face with hearts | smiling hearts</annotation>
+		<annotation cp="🥰">3 | adore | crush | face | heart | hearts | I | love | you | ily | love | romance | smile | smiling</annotation>
 		<annotation cp="🥰" type="tts">smiling face with hearts</annotation>
```

Comment:
'I' by itself seems unexpected, same with '3', I thought, it would be '<3' to get a heart related emoji?

Reply:
Yeah, I think the phrase splitting had a few too many vitamins...

Reply:
Leaving 3 alone.